### ISSUE TITLE
ERROR: [xmldom error]

### ISSUE THREAD

**Issue by renecardozo** [vault-development/react-native-svg-uri#129]
![image](https://user-images.githubusercontent.com/16846689/40551894-2a37be8e-600c-11e8-8373-255a3beee74a.png)

I got this error, is someone with that same error message?. I think that it is related with SVG file got some error but I would like a feedback maybe I'm doing something bad.

**Comment by fniwes** [comment]
Probably the svg is malformed. Could you please share the file?

**Comment by renecardozo** [comment]
Yes, after did some research I found that the problem was related with SVG image. Sorry I forgot mention that early. So I think that it should closed. Thanks for responding.

**Comment by jiangjiatao** [comment]
i have the same question,how did  you solue it? @renecardozo

**Comment by renecardozo** [comment]
@jiangjiatao it was related with the SVG image because  I was using a malformed it.
